charts/cloudnative-pg/values.yaml

```diff
@@ -66,6 +66,14 @@ config:
 # -- Additinal arguments to be added to the operator's args list.
 additionalArgs: []
 
+# -- Array containing extra environment variables which can be templated.
+# For example:
+#  - name: WATCH_NAMESPACE
+#    value: "{{ .Release.Namespace }}"
+#  - name: MY_VAR
+#    value: "mySpecialKey"
+additionalEnvVars: []
```

Comment:
I believe the predominant convention is `additionalEnv`, and my colleagues love conventions.